src/App.jsx

```diff
@@ -14,70 +15,97 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const defaultSelectedGood = 'Jam';
+const emptySelectedGood = '';
+const noGoodsSelectedMessage = 'No goods selected';
+const selectedBackgroundClass = 'has-background-success-light';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(defaultSelectedGood);
+  const isSelectedGoodText = `${selectedGood} is selected`;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const handleSelectedGood = (good) => {
+    setSelectedGood(good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  const handleClearSelection = () => {
+    setSelectedGood(emptySelectedGood);
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const isGoodSelected = selectedGood !== emptySelectedGood;
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  const clearButton = (
+    <button
+      data-cy="ClearButton"
+      type="button"
+      className="delete ml-3"
+      onClick={handleClearSelection}
+    />
+  );
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  const addButton = good => (
+    <button
+      data-cy="AddButton"
+      type="button"
+      className="button"
+      onClick={() => handleSelectedGood(good)}
```

Comment:
will be better use one function for clear and select good, `() => handleSelectedGood(isSelected ? '' : good)`